app/konnect/org-management/okta-idp.md

```diff
@@ -143,11 +121,66 @@ value is present.
 1. From the list of groups in the preview, identify groups that you want to use in
 {{site.konnect_short_name}}. Take note of these groups.
 
+{% endnavtab %}
+{% navtab SAML %}
+### Prepare the Okta application
```

Comment:
Similarly to the generic topic, you can't nest headings inside tabs. Please turn each section into its own OIDC | SAML tab group.